plugin/package.json

```diff
@@ -14,14 +14,17 @@
     "start:terrain": "vite -c vite-configs/location/modals/terrain/web.ts",
     "start:sidebar": "vite -c vite-configs/sidebar/web.ts",
     "start:location": "vite -c vite-configs/location/web.ts",
+    "start:geolocation": "vite -c vite-configs/geolocation/web.ts",
     "build": "run-s typecheck clean build:plugin",
-    "build:plugin": "run-s build:sidebar build:location",
+    "build:plugin": "run-s build:sidebar build:location build:geolocation-web build:geolocation",
```

Comment:
Sorry I think you misunderstood. I was hoping you could move the whole build:geolocation script up here, not add the web build to the `build:plugin` script.
Similar to keeping component props split between variables and functions, I think its cleaner to keep the builds grouped (plugin, then web)